src/envo/envo-edit.owl

```diff
@@ -22555,16 +22555,13 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01000403> "t
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01000403> <http://purl.obolibrary.org/obo/ENVO_01001803>)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01000403> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00000057>))
 
-# Class: <http://purl.obolibrary.org/obo/ENVO_01000404> (manufactured plastic)
+# Class: <http://purl.obolibrary.org/obo/ENVO_01000404> (obsolete manufactured plastic)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://www.merriam-webster.com/dictionary/plastic") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01000404> "A (portion of) plastic is an (portion of) anthropogenic environmental material including any of numerous organic synthetic or processed materials which are primarily composed of thermoplastic or thermosetting polymers of high molecular weight."^^xsd:string)
```

Comment:
Thanks @raissameyer looks like you used the obsoletion plugin - be sure there are no left over axioms on the class, but it looks like those are all gone.

Reply:
I double checked, and indeed, the axioms are gone